packages/design-system/src/components/Breadcrumbs/Breadcrumbs.module.scss

```diff
@@ -0,0 +1,39 @@
+@use '~@talend/design-tokens/lib/tokens';
+
+.breadcrumb {
+	display: block;
+
+	> ul {
```

Comment:
you can use `[role="list"]` on that `ul` instead, to reset its list-style and its margin/padding
👉🏻 https://github.com/Talend/ui/blob/master/packages/design-system/src/components/ThemeProvider/ThemeProvider.style.ts#L6
👉🏻 https://github.com/hankchizljaw/modern-css-reset/blob/master/src/reset.css#L22